New version: TextModels v0.2.0

JuliaRegistrator opened the pull request:
- Registering package: TextModels
- Repository: https://github.com/JuliaText/TextModels.jl
- Created by: @rssdev10
- Version: v0.2.0
- Commit: abeb384a618dd3bede6cb43732bc28587d79eb17
- Git reference: HEAD
- Description: Neural Network based models for Natural Language Processing
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
@JuliaRegistrator register

Release notes:

## Breaking changes
- Major dependency updates: Upgraded core dependencies to their latest major versions (e.g., Flux.jl).
- Model loading compatibility: Some previously pre-trained models may no longer be loadable due to structural or serialization changes.
- Julia version: Based on the Flux.jl dependency, the minimal version of Julia is 1.10 for now.
`````
<!-- END RELEASE NOTES -->

rssdev10 commented:
@JuliaRegistrator register

Release notes:

## Breaking changes
- Major dependency updates: Upgraded core dependencies to their latest major versions (e.g., Flux.jl).
- Model loading compatibility: Some previously pre-trained models may no longer be loadable due to structural or serialization changes.
- Julia version: Based on the Flux.jl dependency, the minimal version of Julia is 1.10 for now.

JuliaHubRegistrator committed:
New version: TextModels v0.2.0

UUID: 77b9cbda-2a23-51df-82a3-24144d1cd378
Repo: https://github.com/JuliaText/TextModels.jl.git
Tree: 0c9ee13f3367d2d342f108bf288b201a5fe6a664

Registrator tree SHA: be588f25c435fa5b456b986adf2f5c02bd3298f2

rssdev10 commented:
continue

[noblock]
